pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -837,6 +901,16 @@ func TestValidateKubeSchedulerConfigurationV1(t *testing.T) {
 			config:         repeatedSchedulerName,
 			errorString:    "Duplicate value",
 		},
+		"greater-than-100-profile-percentage-of-nodes-to-score": {
+			expectedToFail: true,
+			config:         profilePercentageOfNodesToScore101,
+			errorString:    "not in valid range [0-100]",
+		},
+		"negative-profile-percentage-of-nodes-to-score": {
+			expectedToFail: true,
+			config:         profilePercentageOfNodesToScoreNegative,
+			errorString:    "not in valid range [0-100]",
```

Comment:
Do you mind working on a new PR to change `errorString` by `field.ErrorList` so the coverage is more comprehensive?

Reply:
Sure, I can work on it in another PR. Also, replace deprecated `pointer.Int32Ptr` with `pointer.Int32`.